lib/browser.js

```diff
@@ -9,7 +9,7 @@ var after = require('after');
 var utf8 = require('wtf-8');
 
 var base64encoder;
-if (global.ArrayBuffer) {
+if (global&&global.ArrayBuffer) {
```

Comment:
There's no official style guide, but could you add a space here please? (`global && global.ArrayBuffer`)